plugins/patterns/patterns_inspector.rb

```diff
@@ -32,20 +32,25 @@ def inspect(_filter, _options = {})
         inspect_with_tasksel
       else
         @patterns_supported = false
-        @status = "For a patterns inspection please install the package tasksel " \
+        @status = "For a tasks inspection please install the package tasksel " \
```

Comment:
Reading all these / I guess it might make sense to also allow `tasks` as a synonym for `patterns`. But I would ask others if they agree.